lib/dialects/mssql/query-generator.js

```diff
@@ -570,7 +570,14 @@ module.exports = (function() {
 
       if (options.limit || options.offset) {
         if (!options.order || (options.include && !subQueryOrder.length)) {
-          fragment += ' ORDER BY ' + this.quoteIdentifier(model.primaryKeyAttribute);
+          if(!isSubQuery && options.order) {
+            fragment += ', ';
```

Comment:
Is there a test that expresses this line? it looks like it should fail since the fragment you are adding at this point would look like ", pkAttribute"?